pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.alfresco</groupId>
     <artifactId>api-explorer</artifactId>
-    <version>5.2.f-SNAPSHOT</version>
+    <version>5.2.1-SNAPSHOT</version>
```

Comment:
Out of interest, when did we release CS (Repo) 5.2.f and what did it contain ?

I assume there were no related API changes hence we're skipping to 5.2.g ??